src/formatDate.js

```diff
@@ -50,7 +51,50 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let newDate = date;
+
+  if (toFormat[2] === 'YY' && fromFormat[2] === 'YYYY') {
```

Comment:
You can destructure array elements from `fromFormat` and `toFormat` and simplify your code.